vue-components/src/components/TextArea.vue

```diff
@@ -11,10 +11,12 @@
 			</label>
 		</span>
 		<textarea
+			:id="id"
 			:class="[
 				'wikit-TextArea__textarea',
 				`wikit-TextArea__textarea--${resizeType}`
 			]"
+			:value="value"
```

Comment:
Is this textarea value (as well as the corresponding test) actually a change that should go into #459? Or is it somehow related to the story?

Reply:
I don't think it really relates to any of them, I only noticed it since I was looking at the parent story while working where it is mentioned but not mentioned by any of the sub tickets we had to implement. I can separate this to a followup smaller PR though if you think that's better.

Reply:
no, if it doesn't obviously belong in #459, I think you can just as well leave it in here.